src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/AzureVmModels/AzureRmRecoveryServicesAzureVmItem.cs

```diff
@@ -20,7 +20,57 @@
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models
 {
+    /// <summary>
+    /// Represents IaaSVM Item Class
+    /// </summary>
     public class AzureRmRecoveryServicesIaasVmItem : AzureRmRecoveryServicesItemBase
+    {   
+        public string VirtualMachineId { get; set; }
+
+        /// <summary>
+        /// Protection Status of the item
+        /// </summary>
+        public string ProtectionStatus { get; set; }
+
+        /// <summary>
+        /// Protection State of the item
+        /// </summary>
+        public string ProtectionState { get; set; }
```

Comment:
This is as per service contract. In case, if we this property is not implemented by some new provider